client/src/components/Dashboard/Dashboard.jsx

```diff
@@ -9,29 +9,47 @@ import Sysmon from '../SystemMonitor';
 import * as ROUTES from '../../constants/routes';
 import './Dashboard.global.css';
 import styles from './Dashboard.module.css';
+import CustomDashboardSection from './CustomDashboardSection';
+
+function getPanelCount(standardApps = [], optionalApps = [], customApps = []) {
+  return standardApps.length + optionalApps.length + customApps.length;
+}
 
 function Dashboard() {
-  const hideApps = useSelector((state) => state.workbench.config.hideApps);
+  const { hideApps, hideManageAccount, customDashboardSection } = useSelector(
+    (state) => state.workbench.config
+  );
   const { hideSystemMonitor } = useSelector((state) => state.systemMonitor);
+  const panelCount = getPanelCount(
+    ['DashboardTickets'],
+    [hideApps, hideSystemMonitor].filter((isHidden) => !isHidden),
+    ...(Boolean(customDashboardSection) ? [['customDashboardSection']] : [])
+  );
+
   return (
     <Section
       bodyClassName="has-loaded-dashboard"
       messageComponentName="DASHBOARD"
       messages={<BrowserChecker />}
       header="Dashboard"
       headerActions={
-        <Link to={`${ROUTES.WORKBENCH}${ROUTES.ACCOUNT}`} className="wb-link">
-          Manage Account
-        </Link>
+        !hideManageAccount && (
+          <Link to={`${ROUTES.WORKBENCH}${ROUTES.ACCOUNT}`} className="wb-link">
+            Manage Account
+          </Link>
+        )
       }
-      contentClassName={styles['panels']}
-      contentLayoutName="twoColumn"
+      contentClassName={`${styles['panels']} count--${panelCount}`}
+      contentLayoutName={hideApps ? 'balance' : 'twoColumn'}
```

Comment:
I messed up merge conflict resolution in [#689](https://github.com/TACC/Core-Portal/pull/689). I only noticed after your merged it. These can be Unequal now. I tested, but you probably should also.

```suggestion
      contentLayoutName={hideApps ? 'balanceUnequal' : 'twoColumnUnequal'}
```